msteams-platform/concepts/design/personal-apps.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Designing your personal app
-description: Learn how to design a Teams personal app and get the Microsoft Teams UI Kit, create components, such as, dashboard, form, task board for Mobile and Desktop experience. Learn the best practices for developing personal apps.
+description: Learn how to design a Teams personal app and get the Microsoft Teams UI Kit, create components, such as, dashboard, form, task board for Mobile and Desktop experience.
```

Comment:
The first sentence reads something like, Learn how to get UI kit. Needs an edit.

Small case for mobile and desktop.